XAMLator.Server.Shared/IdeIP.txt

```diff
@@ -1 +1 @@
-﻿127.0.0.1
```

Comment:
This shouldn't be committed. The weaving process should change the IP automatically with your development's machine one, if that's not working can you please open a new bug for it?